pages/categories/downstream.md

```diff
@@ -10,12 +10,97 @@ Downstream emissions (Category D) are the emissions produced by using an organis
+
+
+{% include linkedHeading.html heading="End-User Device Embodied Carbon" level=3 %}
+
+It's important to consider the embodied carbon of the device that goes beyond just the operational emissions. This includes the carbon emissions generated from the mining of raw materials, production of the device, transport and end-of-life processing. This data can usually be obtained from the manufacturers, for example:
+
+- [Apple Environmental Reports](https://www.apple.com/environment/)
+- [Dell Product Carbon Footprints](https://www.dell.com/en-uk/dt/corporate/social-impact/advancing-sustainability/climate-action/product-carbon-footprints.htm)
+- [HP Product Carbon Footprints](https://h20195.www2.hp.com/v2/library.aspx?doctype=95&footer=95&filter_doctype=no&showregionfacet=yes&filter_country=no&cc=us&lc=en&filter_oid=no&filter_prodtype=rw&prodtype=ij&showproductcompatibility=yes&showregion=yes&showreglangcol=yes&showdescription=yes%23doctype-95&sortorder-popular&teasers-off&isRetired-false&isRHParentNode-false&titleCheck-false#doctype-95&sortorder-popular&teasers-off&isRetired-false&isRHParentNode-false&titleCheck-false)
+- [Samsung Life Cycle Assessment](https://www.samsung.com/global/sustainability/focus/products/sustainability-in-our-products/)
+
+Typically, a laptop has on average, an expected life span of four years. Approximately 80% of its product life cycle emissions are attributed to production emissions and 20% to operational emissions. Transport and end-of-life processing make up negligible percentages. It is important to factor in the average life cycles of devices, given that the majority of its carbon emissions are attributed to production. Extending the life of a device, i.e. using it beyond its expected life cycle bears a significant impact on the hardware efficiency. Read more about [amortisation here](/glossary#amortisation).
+
+End-user device life spans are driven by several factors, but it is often driven by social behaviour (a market-driven desire for newer, improved technology). Other factors include:
+- built-in obsolescence by manufacturers - removed support and security 
+- applications demanding higher performance, the latest operating systems and hardware
+- un-repairable devices and hardware components (cannot physically be repaired/replaced or is cost prohibitive to do so)
+
+Whilst the end-user device's embodied carbon is beyond the direct control of an organisation's products and services, there are several things to consider, that can help extend the life of these devices:
+- ensure backward compatibility with hardware, operating systems and browsers, for as long as reasonably possible.
+- optimise applications; demand less from hardware and resources, minimise data transfer.
```

Comment:
Previously, you haven't used full stops at the end of bullet points; I'd suggest removing them here.